.github/bin/dist-nuget.sh

```diff
@@ -20,12 +20,14 @@ if [ "$NUGET_API_KEY" = "" ]; then
 fi
 
 # Publish a package only if the repository is upstream (planetarium/libplanet)
-# and the branch is for releases (master or maintenance-*).
+# and the branch is for releases (master or *-maintenance or 9c-*).
 # shellcheck disable=SC2235
-if [ "$GITHUB_REPOSITORY" != "planetarium/libplanet" ] || (
-    [ "$GITHUB_REF" = "${GITHUB_REF#refs/tags/}" ] &&
-    [ "$GITHUB_REF" != refs/heads/master ] &&
-    [ "$GITHUB_REF" = "${GITHUB_REF#refs/heads/maintenance-}" ] ); then
+if [ "$GITHUB_REPOSITORY" != "planetarium/libplanet" ] || ! [[ \
```

Comment:
You mean like this?

```suggestion
if ! [[ \
    "$GITHUB_REPOSITORY" = "planetarium/libplanet" || \
```

Reply:
it's close. actually, I imagined as below

```bash
if [ "$GITHUB_REPOSITORY" == "planetarium/libplanet" ] && [[<<other checks>>]]
   function dotnet-nuget {
    dotnet nuget "$@"	    dotnet nuget "$@"
  }	  
}
else
  function dotnet-nuget {
    echo "DRY-RUN: dotnet nuget" "$@"	    echo "DRY-RUN: dotnet nuget" "$@"
  }
fi
```

Reply:
@longfin Amended!